### PR TITLE
Add “How it works” section to README

### PR DESCRIPTION
**Pull request by 343dev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.1.1 (12.08.2020)
```
Comment (igoradamenko) [inline CHANGELOG.md:3]: Nah, let's not bump the version. The PR changes README only, there're no bugfixes.

**Comment by igoradamenko** [comment]
@goganchic, approve?

File: CHANGELOG.md (hunk 1 above)
Comment (343dev) [reply CHANGELOG.md]: Done.

Comment (igoradamenko) [reply CHANGELOG.md]: 🚀

**Pull request merged**